examples/pytorch/text-classification/run_glue_no_trainer.py

```diff
@@ -327,6 +327,9 @@ def main():
     tokenizer = AutoTokenizer.from_pretrained(
         args.model_name_or_path, use_fast=not args.use_slow_tokenizer, trust_remote_code=args.trust_remote_code
     )
+    tokenizer.pad_token = tokenizer.eos_token
+    config.pad_token_id = tokenizer.pad_token_id
```

Comment:
We shouldn't override the pad_token if it's already set

```suggestion
    if tokenizer.pad_token is None: 
        tokenizer.pad_token = tokenizer.eos_token
    config.pad_token_id = tokenizer.pad_token_id
```

Reply:
that makes sense, sure ill update it